go.sum

```diff
@@ -832,8 +832,9 @@ golang.org/x/crypto v0.0.0-20201002170205-7f63de1d35b0/go.mod h1:LzIPMQfyMNhhGPh
 golang.org/x/crypto v0.0.0-20210220033148-5ea612d1eb83/go.mod h1:jdWPYTVW3xRLrWPugEBEK3UY2ZEsg3UU495nc5E+M+I=
 golang.org/x/crypto v0.0.0-20210817164053-32db794688a5/go.mod h1:GvvjBRRGRdwPK5ydBHafDWAxML/pGHZbMvKqRZ5+Abc=
 golang.org/x/crypto v0.0.0-20210921155107-089bfa567519/go.mod h1:GvvjBRRGRdwPK5ydBHafDWAxML/pGHZbMvKqRZ5+Abc=
-golang.org/x/crypto v0.0.0-20220214200702-86341886e292 h1:f+lwQ+GtmgoY+A2YaQxlSOnDjXcQ7ZRLWOHbC6HtRqE=
```

Comment:
@aravindhp it looks like some of our dependencies are using even older versions of this library. Does that need to be addressed as well?

Reply:
This same version is also being used elsewhere as well

Reply:
I don't think that's needed. In fact we are actually not affected by the CVE because we don't match the [conditions](https://groups.google.com/g/golang-announce/c/-cp44ypCT5s/m/wmegxkLiAQAJ) for the crash to occur. I bumped the versions to be on the safe side. LMK if you think I am misreading the CVE.

Reply:
Sounds good, just making sure